po/es.po

```diff
@@ -4875,104 +4950,107 @@ msgstr ""
 #: modules/devices/x86/x86_data.c:143
 msgctxt "x86-flag"
 msgid "Running on a hypervisor"
-msgstr "Corriendo en un hipervisor"
+msgstr "Ejecutándose en un hipervisor"
 
 #. /Cyrix/Centaur-defined CPU features, CPUID level 0xC0000001
 #. /flag:rng
 #: modules/devices/x86/x86_data.c:145
 msgctxt "x86-flag"
 msgid "Random Number Generator present (xstore)"
-msgstr "Generador de numeros aleatorios presente (xstore)"
+msgstr "Generador de números aleatorios presente (xstore)"
 
 #. /flag:rng_en
 #: modules/devices/x86/x86_data.c:146
 msgctxt "x86-flag"
 msgid "Random Number Generator enabled"
-msgstr "Generador de numeros aleatorios activo"
+msgstr "Generador de números aleatorios habilitado"
 
 #. /flag:ace
 #: modules/devices/x86/x86_data.c:147
 msgctxt "x86-flag"
 msgid "on-CPU crypto (xcrypt)"
-msgstr "encriptacion en-CPU (xcrypt)"
+msgstr "Cifrado en CPU (xcrypt)"
 
 #. /flag:ace_en
 #: modules/devices/x86/x86_data.c:148
 msgctxt "x86-flag"
 msgid "on-CPU crypto enabled"
-msgstr "encriptacion en-CPU activada"
+msgstr "Cifrado en CPU habilitado"
 
 #. /flag:ace2
 #: modules/devices/x86/x86_data.c:149
 msgctxt "x86-flag"
 msgid "Advanced Cryptography Engine v2"
-msgstr "Manejador Encriptamiento Avanzado v2"
+msgstr "Motor de criptografía avanzada v2"
 
 #. /flag:ace2_en
 #: modules/devices/x86/x86_data.c:150
 msgctxt "x86-flag"
 msgid "ACE v2 enabled"
-msgstr "ACE v2 presente"
+msgstr "ACE v2 habilitado"
 
 #. /flag:phe
 #: modules/devices/x86/x86_data.c:151
 msgctxt "x86-flag"
 msgid "PadLock Hash Engine"
-msgstr "Manejador PadLock Hash"
+msgstr ""
 
 #. /flag:phe_en
 #: modules/devices/x86/x86_data.c:152
 msgctxt "x86-flag"
 msgid "PHE enabled"
-msgstr "PHE presente"
+msgstr "PHE habilitado"
 
 #. /flag:pmm
 #: modules/devices/x86/x86_data.c:153
 msgctxt "x86-flag"
 msgid "PadLock Montgomery Multiplier"
-msgstr "PadLock Montgomery Multiplier"
+msgstr ""
 
 #. /flag:pmm_en
 #: modules/devices/x86/x86_data.c:154
 msgctxt "x86-flag"
 msgid "PMM enabled"
-msgstr "PMM presente"
+msgstr "PMM habilitado"
 
 #. /flag:lahf_lm
 #: modules/devices/x86/x86_data.c:156
+#, fuzzy
 msgctxt "x86-flag"
 msgid "Load AH from Flags (LAHF) and Store AH into Flags (SAHF) in long mode"
-msgstr "Carga AH desde Flags (LAHF) y almacena AH en Flags (SAHF) en modo long"
+msgstr ""
+"Carga AH desde indicadores (LAHF) y almacena AH en indicadores (SAHF) en modo"
+" largo"
 
 #. /flag:cmp_legacy
 #: modules/devices/x86/x86_data.c:157
 msgctxt "x86-flag"
 msgid "If yes HyperThreading not valid"
-msgstr "Si yes/si HyperThreading no valido"
+msgstr "En caso afirmativo, HyperThreading no es válido"
 
 #. /flag:svm
 #: modules/devices/x86/x86_data.c:158
 msgctxt "x86-flag"
 msgid "\"Secure virtual machine\": AMD-V"
-msgstr "\"Maquina Virtual Segura\": AMD-V"
+msgstr "«Maquina virtual segura»: AMD-V"
```

Comment:
it seems maybe in multilang environment will have some unexpected results.. so for sure.. let them as original with ´\"%s\"´ and in future version next development release make a new pull request with changed to quotation